src/v3/src/constants/idxConstants.ts

```diff
@@ -192,11 +179,18 @@ export const STEPS_MISSING_RELATES_TO: string[] = [
   IDX_STEP.ENROLLMENT_CHANNEL_DATA,
 ];
 
+// export const CHALLENGE_INTENT_TO_I18KEY: Record<string, string> = {
+//   AUTHENTICATION: 'next.idx.return.link.otponly.enter.code.on.page.sigin.in',
+//   RECOVERY: 'next.idx.return.link.otponly.enter.code.on.page.password.reset',
+//   UNLOCK_ACCOUNT: 'next.idx.return.link.otponly.enter.code.on.page.account.unlock',
+//   ENROLLMENT: 'next.idx.return.link.otponly.enter.code.on.page.registration',
+// };
+
 export const CHALLENGE_INTENT_TO_I18KEY: Record<string, string> = {
-  AUTHENTICATION: 'next.idx.return.link.otponly.enter.code.on.page.sigin.in',
-  RECOVERY: 'next.idx.return.link.otponly.enter.code.on.page.password.reset',
-  UNLOCK_ACCOUNT: 'next.idx.return.link.otponly.enter.code.on.page.account.unlock',
-  ENROLLMENT: 'next.idx.return.link.otponly.enter.code.on.page.registration',
+  AUTHENTICATION: 'idx.return.link.otponly.enter.code.on.page.sign.in',
```

Comment:
possible to include these keys in i18nTransformer?

Reply:
This label is for a description element (flow specific), the i18n transformer manages transforming labels for generic elements that is sent from the API. This element is not identified in the API at all it is only added in the custom transformer for the Email Magic Link OTP flow.

Or do you mean to add this constant to the i18nTransformer file?

Reply:
yea, I mean the later one, possible to manage all keys in a centralized place?